File: src/main/resources/archetype-resources/src/main/java/controller/TestController.java
package ${groupId}.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

/**
 * @author wangxiaodong
 * Created by wangxiaodong on 2018/6/6.
 */
@RestController
public class TestController {


    @GetMapping(value = "/test")
    String index(int id){
        return "test";
    }


}

File: src/main/resources/archetype-resources/src/main/kotlin/controller/TestController.kt
package ${groupId}.controller

import com.github.lyrric.lianjia.service.*
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import javax.annotation.Resource


@RestController
@RequestMapping(value = ["/"])
class TestController{

    /**
     * 周统计
     */
    @GetMapping(value = ["/test"])
    fun test():String{
        return "test"
    }




}